DOCSP-29461: node lab embedding (#820)
mongodb/docs-node: source/fundamentals/crud/read-operations/retrieve.txt

--------

You can use read operations to retrieve data from your MongoDB database.

> To request results based on a set of criteria from the existing set of
> data, you can use a find operation such as the ``find()`` or

Reviewer: S: not necessary, but I feel like saying "existing collection" is easier to understand than "existing set of data"(and avoids repeating "set")
```suggestion
To request results based on a set of criteria from an existing collection, you can use a find operation such as the ``find()`` or
```

> ``findOne()`` methods.

Author: noticed this too and changed it!

.. tip:: Interactive Lab
   
   You can complete an interactive lab that uses the
   ``find()`` method to retrieve data in the
   :ref:`node-retrieve-instruqt-lab` section of this guide.

> You can also further specify the information you are requesting by

Reviewer: S: To avoid the present continuous tense
```suggestion
You can also further specify the information that the find operation returns
```

      :start-after: start watch crud example
      :end-before: end watch crud example

For a runnable example of the ``watch()`` method using the NodeJS driver, see
the :ref:`change streams <node-usage-watch>` usage example.

.. _node-retrieve-instruqt-lab:

> Interactive Find Operation Lab

Author: I'll update all the headings in these pages to be action oriented.

------------------------------

This lab helps you understand how to perform read operations in MongoDB

> by using the ``find()`` method. You do not need to install
> MongoDB or a code editor to complete the lab.

Reviewer: S: since Vale doesn't like "need," you could reword to something like:
```suggestion
by using the ``find()`` method. You can complete this lab without installing
MongoDB or a code editor.
```

Author: changed in all

> .. instruqt:: /mongodb-docs/tracks/find-node?token=em_OVNHWCPNPMLwNOCm

Author: good point!
